official-antistasi-community/A3-Antistasi
# 🔠 log calls code for remote logging.

## What type of PR is this.
1. [x] Bug
* Change
* Enhancement

### What have you changed and why?

* 🔠 log calls code for remote logging.

### Please verify the following and ensure all checks are completed.


### Is further testing or are further changes required?
1. [x] No
* Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
1. Use the Arma 3 Launcher to start a dedicated server.
2. use `[1, "or thank vacation", "very filed"] call A3A_fnc_log;`
3. Check that there is no serialisation warning
